downstream/snippets/snip-gw-authentication-next-steps.adoc

```diff
@@ -0,0 +1 @@
+To control which users are allowed into the {PlatformNameShort} server, and placed into {PlatformNameShort} organizations or teams based on their attributes (like username and email address) or to what groups they belong, continue to xref:gw-mapping[Mapping].
```

Comment:
Do we need to specify {PlatformNameShort} twice? Can we remove the attribute before 'organizations' or say something like Ansible organizations, to shorten the sentence? This is a suggestion, so please update only if you deem it necessary.

Reply:
I'm going to leave this one because I was asked to be specific for both server and organizations by an SME.